src/components/challenges/active.js

```diff
@@ -1,6 +1,7 @@
 import { useState } from 'react';
 import classNames from './styles.module.scss';
-import Label from '../label/label';
+import Details from './details/details';
```

Comment:
Nit: Avoid pattern like this, it's redundant and adds no major value
I personally prefer: `./details` (/details/index.js)

Reply:
Ok Got it, in my next PR will fix this.